noir_stdlib/src/collections/bounded_vec.nr

```diff
@@ -121,6 +121,17 @@ impl<T, MaxLen> BoundedVec<T, MaxLen> {
         }
         ret
     }
+
+    pub fn map<Env>(self, f: fn[Env](T) -> T) -> Self {
```

Comment:
The type can be more general here:
```suggestion
    pub fn map<U, Env>(self, f: fn[Env](T) -> U) -> BoundedVec<U, MaxLen> {
```

Reply:
Ah you're absolutely right, I totally missed this. Will also add a test that changes the type. Thanks!